### PR TITLE
Updated Description for PIMS_PPH_STATUS_TYPE

### PR DESCRIPTION
**Pull request by dfilteau**
Closing parenthesis added to end of COMBO description

### CODE REVIEW

File: database/mssql/scripts/dbscripts/PSP_PIMS_S31_01/Alter Down/87_DML_PIMS_PPH_STATUS_TYPE_Edit.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+SET @CurrCd = N'COMBO';
+
+SELECT PPH_STATUS_TYPE_CODE
+FROM   PIMS_PPH_STATUS_TYPE
+WHERE  PPH_STATUS_TYPE_CODE = @CurrCd;
+
+IF @@ROWCOUNT = 1
+  BEGIN
+  UPDATE PIMS_PPH_STATUS_TYPE 
+  SET    DESCRIPTION                = N'Combination (PPH & Non-PPH)'
```
Comment (FuriousLlama) [inline database/mssql/scripts/dbscripts/PSP_PIMS_S31_01/Alter Down/87_DML_PIMS_PPH_STATUS_TYPE_Edit.sql:21]: Should the alter down revert the change? This one looks the same as the Alter up

Comment (dfilteau) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S31_01/Alter Down/87_DML_PIMS_PPH_STATUS_TYPE_Edit.sql:21]: The scripts will repair the problem, no matter what, so I put them in both.

**Review by FuriousLlama — APPROVED** [review]
